java/lib/edge/src/main/java/org/eclipse/tahu/edge/EdgeNodeMetricMaps.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Licensed Materials - Property of Cirrus Link Solutions
+ * Copyright (c) 2023 Cirrus Link Solutions LLC - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
```

Comment:
This doesn't sound right?

Reply:
Thanks - fixed.

Reply:
FWIW looks like there are 3 other files in the repo with a similar issue